src/orb.yml.hbs

```diff
@@ -796,6 +796,7 @@ commands:
             if [ -n "${CCI_ORB_AWS_ECS_PROXY_CONFIGURATION}" ] && [ "${CCI_ORB_AWS_ECS_PROXY_CONFIGURATION}" != "{}" ]; then
                 set -- "$@" --proxy-configuration "${CCI_ORB_AWS_ECS_PROXY_CONFIGURATION}"
             fi
+            echo "Before register-task-definition \$@ is: $@"
```

Comment:
Apologies that I didn't mention this in my earlier review, but I would like for the debug output to be toggleable by a new parameter, disabled by default. The parameter should also be added to the jobs that call the affected commands.

Another thought I have is, we could log the entire AWS command instead of just "$@" . Would that be more helpful for debugging?

For example (assuming the new parameter is called `show-command`)

```
<<# parameters.show-command>>set -x<</ parameters.show-command>>
REVISION=$(aws ecs register-task-definition ...)
<<# parameters.show-command>>set +x<</ parameters.show-command>>
```